src/Microsoft.DotNet.ProjectModel/ProjectReaderSettings.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    public class ProjectReaderSettings
+    {
+        public string VersionSuffix = null;
```

Comment:
Also, it makes turning into a property later a breaking change. Best practices are to always use properties for public data members, and we have to consider that since we are a de-facto source of development practices for other C# developers by the fact we have "dotnet" in our GitHub Org ;)